README.md

```diff
@@ -1,169 +1,96 @@
 # kube-spawn
 
-## Introduction
-
-__kube-spawn__ is a tool for creating a multi-node Kubernetes cluster
+`kube-spawn` is a tool for creating a multi-node Kubernetes cluster
 on a single machine, created mostly for developers __of__ Kubernetes.
 
 It aims to be as similar to the solutions recommended for production
 clusters as possible.
 
-## Getting started
+## Requirements
 
-### Build proper systemd-nspawn version
+* **Host:**
+  - `systemd-nspawn` at least version 233
+  - tested: Fedora 26, ...
 
-Unfortunately, there is [one pending feature](https://github.com/systemd/systemd/pull/4395)
-of systemd-nspawn which is merged in master, but not released yet.
-You will need to build your own systemd-nspawn binary.
+* **Kubernetes** at least version 1.7.0
 
-We highly recommend using CoreOS' fork which backported that feature
-to the 231 version of systemd (which is the one that Fedora and
-the other popular distributions are using in its stable releases).
+## Quickstart
 
-In order to do that, please use the following commands:
+To quickly test `kube-spawn`, the easiest (and tested) way is to use the prepared [Vagrant setup](doc/vagrant.md).
```

Comment:
Wording wise, I'd suggest....

kube-spawn should run well on Fedora 26, ..., etc. If you want to test in a controlled environment, you can test it using [Vagrant](doc/vagrant.md).